controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -760,8 +760,9 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(Socket *sd,
 
     tlvRuid->radio_uid() = network_utils::mac_from_string(ruid);
 
-    const auto &bss_info_confs = database.get_bss_info_configuration(al_mac);
-    uint8_t num_bsss           = 0;
+    const auto &bss_info_confs =
+        database.get_bss_info_configuration(network_utils::mac_from_string(al_mac));
```

Comment:
Use `tlvwscM1->mac_attr().data` directly instead of converting back from string.

Reply:
yes, I have noticed it late but didn't push yet, I'll upload soon :)